plugins/cl-who-renderer.lisp

```diff
@@ -0,0 +1,22 @@
+(eval-when (:compile-toplevel :load-toplevel)
+  (asdf:load-system 'cl-who))
```

Comment:
`ql:quickload` should be used instead. Otherwise if the user hasn't first quickloaded cl-who once this will fail instead of fetching the dependency.